Sources/Sentry/Public/SentryOptions.h

```diff
@@ -766,6 +766,29 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, readonly) SentryExperimentalOptions *experimental;
 
+#if TARGET_OS_IOS && SENTRY_HAS_UIKIT
+
+/**
+ * Block used to configure the user feedback widget, form, behaviors and submission data.
+ */
+API_AVAILABLE(ios(13.0))
+typedef void (^SentryUserFeedbackConfigurationBlock)(
+    SentryUserFeedbackConfiguration *_Nonnull configuration);
+
+/**
+ * A block that can be defined that receives a user feedback configuration object to modify.
+ * @warning This is an experimental feature and may still have bugs.
+ * @note This is unrelated to @c SentrySDK.captureUserFeedback which is the old method of submitting
+ * user feedback you've already gathered via your own UI
+ * (see https://docs.sentry.io/platforms/apple/user-feedback/#user-feedback-api). The new method
+ * uses either this block to configure and a widget and UI form to gather feedback, or directly
```

Comment:
`l`:
```suggestion
 * (see https://docs.sentry.io/platforms/apple/user-feedback/#user-feedback-api). This method
 * uses either this block to configure and a widget and UI form to gather feedback, or directly
```

Reply:
i reworded it a little differently because the "the new method" you wanted to change didn't refer to an actual function/method to call, but rather, an entire strategy of how to gather feedback: https://github.com/getsentry/sentry-cocoa/blob/ae42d842ef5367a85ca99fd3564cef4a4d8c8e75/Sources/Sentry/Public/SentryOptions.h#L771-L782